docs/governance/rfc/005-python-implementation.md

```diff
@@ -0,0 +1,127 @@
+# 005: Python Implementation
+
+I propose an official Python implementation of Connect, based on the prototype at https://github.com/firetiger-oss/connect-python/.
+This should include both the client and server side.
+
+## Why
+
+Python is a major language used heavily in web development.
+Adding an official implementation will help the Connect community focus its efforts.
+
+## Anticipated Complications
+
+
+The Python ecosystem has a big split between asynchronous and synchronous I/O, which will complicate the implementation story.
+Supporting both is important; Python applications tend to commit to one of the two sides of the split and have difficulty using the other.
+
+Further complicating matters is the Python community is divided regarding the right asynchronous implementation.
+[asyncio](https://docs.python.org/3/library/asyncio.html) is the standard library's answer, but you can find adherents of [https://trio.readthedocs.io/en/stable/](trio) and other asynchronous event loops.
```

Comment:
```suggestion
[asyncio](https://docs.python.org/3/library/asyncio.html) is the standard library's answer, but you can find adherents of [trio](https://trio.readthedocs.io/en/stable/) and other asynchronous event loops.
```

To fix the broken link. Otherwise very excited to see this adopted 🥳